### PR TITLE
Fixed the test sse-s3 cleanup issue as per the new changes made in 2.7.0

### PR DESCRIPTION
**Pull request by sn-px**
**What this PR does / why we need it**:
Minor fix to move deleting restore object before cluster delete is performed

**Which issue(s) this PR fixes** (optional)
Closes #https://portworx.atlassian.net/browse/PB-5873

**Special notes for your reviewer**:
https://aetos.pwx.purestorage.com/resultSet/testSetID/561802
https://jenkins.pwx.dev.purestorage.com/job/Users/job/santoshn/job/sse-byoc/16/consoleFull

### CODE REVIEW

**Review by kshithijiyer-px — APPROVED** [review]
LGTM

File: tests/backup/backup_sse_test.go [hunk 1]
```diff
@@ -515,16 +515,16 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
...
-		log.Infof("Deleting backup schedule policy")
-		err = DeleteSchedule(scheduleName, SourceClusterName, BackupOrgID, ctx)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Verification of deleting backup schedule - %s", scheduleName))
-		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 		// Delete restores
 		for _, restoreName := range restoreList {
 			err = DeleteRestore(restoreName, BackupOrgID, ctx)
 			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting restore [%s]", restoreName))
 		}
+		// Delete backup schedule policy
```
Comment (ak-px) [inline tests/backup/backup_sse_test.go:523]: nit: schedule

**Review by ak-px — APPROVED** [review]
LGTM

File: tests/backup/backup_sse_test.go [hunk 2]
```diff
@@ -515,16 +515,16 @@ var _ = Describe("{CreateBackupAndRestoreForAllCombinationsOfSSES3AndDenyPolicy}
...
-		// Delete backup schedule policy
-		log.Infof("Deleting backup schedule policy")
-		err = DeleteSchedule(scheduleName, SourceClusterName, BackupOrgID, ctx)
-		dash.VerifySafely(err, nil, fmt.Sprintf("Verification of deleting backup schedule - %s", scheduleName))
-		CleanupCloudSettingsAndClusters(backupLocationMap, credName, cloudCredUID, ctx)
 		// Delete restores
 		for _, restoreName := range restoreList {
 			err = DeleteRestore(restoreName, BackupOrgID, ctx)
 			dash.VerifySafely(err, nil, fmt.Sprintf("Deleting restore [%s]", restoreName))
 		}
```
Comment (mkoppal-px) [inline tests/backup/backup_sse_test.go:522]: I think we dont have to cleanup the restores now right? Deleting the cluster will delete restores

Comment (kshithijiyer-px) [reply tests/backup/backup_sse_test.go:522]: Why not keep it to maintain backward compatibility?

**Review by kphalgun-px — APPROVED** [review]
LGTM

**Review by mkoppal-px — APPROVED** [review]
LGTM

**Pull request merged**